### DISCUSSION TITLE
Blog Post: Deep Dive into Custom Types

### DISCUSSION THREAD

**Discussion by altugbakan** [stellar/sorobanathon#97] [General]
Hey!

I have written a technical blog post about the Custom Types quest. It is about helping developers implement the custom structs and enums on Rust, and build the custom JSON objects. You can check it out [here](https://dev.to/altug/soroban-quest-custom-types-5d54), and take your time, as this quest was pretty long.

I hope you like it!

**Comment by achallagundla** [comment]
Heya,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1200 XLM (a small token of gratitude for all your help with SQ)! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
